app/templates/gentelella/guest/home.html

```diff
@@ -18,8 +18,11 @@
         }
 
         .event-holder img {
-            width: 100%;
-            height: 165px;
```

Comment:
> Instead of all this 

To be clear, I meant , keep the existing width (100%) and height (165px) and just add the above style. 😄